### DISCUSSION TITLE
4 port emulation?

### DISCUSSION THREAD

**Discussion by navas** [Anime4000/RTL960x#332] [General]
hello everyone,

I use my ISP's modem as bridge mode to connect to my openwrt router. the modem has 4 lan ports and I can setup pppoe in all 4 ports and combine the speed using mwan3. Is it possible with  DFP-34X-2C2 to do the same ? I am on 220923. I can create multiple pppoe connections but all seems sharing vlan 100 (eth2.100) so I am not getting speed multiplication. Is there any way I can do this with this XPON?

**Comment by Anime4000** [comment]
first, you need a 2500 Base-X SFP support, so ODI can use 2.5Gbps Link Speed

second, for multiple PPPoE on same interface, but ISP need different MAC Address, you can use VRRP tricks:
sfp1 > vlan100 > vrrp1 > pppoe1
sfp1 > vlan100 > vrrp2 > pppoe2
...
sfp1 > vlan100 > vrrpX > pppoeX

**Reply by navas** [reply]
thank you! I am not familiar with VRRP protocol, would you please kind enough to give some reference where I can get started?

**Reply by Anime4000** [reply]
tldr; VRRP just another child interface that run on top main one
![image](https://github.com/user-attachments/assets/1df94342-813e-4439-9c20-6481ed337f72)
In mikrotik, creating multiple VRRP on top of main interface it create different mac based on the ID

see if OpenWRT has it

**Reply by navas** [reply]
Hi

Thank you for your help. But it turns out, All I need is a simple macvlan trick with a different MAC ID's, and then it magically worked!

**Reply by Anime4000** [reply]
alright, nice
